csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -7,7 +7,7 @@
     <VersionPrefix>3.7.0</VersionPrefix>
     <LangVersion>6</LangVersion>
     <Authors>Google Inc.</Authors>
-    <TargetFrameworks>netstandard1.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.1;net45</TargetFrameworks>
```

Comment:
This would be considered a breaking change no? Adding another target for netstandard2.0 (or netstandard 1.1 if you want) would probably be better here.

Reply:
+1